src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -4,10 +4,10 @@
 public class Task3 {
     public static String comboString(String s1, String s2) {
         if (s1 == null) {
-            s1 = String.valueOf(s1);
-        }
+            return String.valueOf(s1)+s2+String.valueOf(s1);
+                    }
         if (s2 == null) {
-            s2 = String.valueOf(s2);
+            return String.valueOf(s2)+s1+String.valueOf(s2);
```

Comment:
there is no need for `valueOf` method usage, cause if you concat anything with String - the result will be String.